### PR TITLE
Skip gpu tests in regions without gpu

### PR DESCRIPTION
**Pull request by yangaws**
*Issue #, if available:*

*Description of changes:*
In us-west-1, eu-west-2 and ca-central-1, there's no gpu ec2 instance like P2/P3. So the tf tests using gpu instance will fail and we should skip them in these regions.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integ/test_tf_cifar.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def __call__(self, data):
 
 @pytest.mark.continuous_testing
 @pytest.mark.skipif(PYTHON_VERSION != 'py2', reason="TensorFlow image supports only python 2.")
+@pytest.mark.skipif(REGION in ['us-west-1', 'eu-west-2', 'ca-central-1'])
```
Comment (laurenyu) [inline tests/integ/test_tf_cifar.py]: add a reason. same below.

**Comment by nadiaya** [comment]
Other integ tests also run gpu tests (e.g. pytorch)

**Comment by yangaws** [comment]
@nadiaya Only need to skip the tests that marked as continuous_testing (run as canary). For other p2 tests like pytorch async, it's not part of canary. But I guess I can still make the change though.

File: tests/integ/test_tf_cifar.py (hunk 1 above)
Comment (yangaws) [reply tests/integ/test_tf_cifar.py]: Updated

File: tests/integ/test_chainer_train.py [hunk 2]
```diff
@@ -35,6 +35,8 @@ def test_distributed_cpu_training(sagemaker_session, chainer_full_version):
     _run_mnist_training_job(sagemaker_session, "ml.c4.xlarge", 2, chainer_full_version)
 
 
+@pytest.mark.skipif(REGION in ['us-west-1', 'eu-west-2', 'ca-central-1'],
+                    reason='No ml.p2.xlarge supported in these regions')
```
Comment (nadiaya) [inline tests/integ/test_chainer_train.py]: what about p3? can't we use p3 then?

Comment (yangaws) [reply tests/integ/test_chainer_train.py]: P3 is either not supported or supported only by 1/3 or availability zones. So for these 3 regions we just skip tests for now to make canary good for now.

**Pull request merged**